### PR TITLE
Add OpenLieroX

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://sourceforge.net/p/openlierox/mailman/openlierox-devel/thread/bd1fc9e5-7201-f137-f075-cda8d372edc8%40mailaender.name/#msg37659349
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* ~~https://sourceforge.net/p/openlierox/code/merge-requests/1/~~ https://github.com/albertz/openlierox/pull/842

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build net.sourceforge.OpenLieroX

File: net.sourceforge.OpenLieroX.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">net.sourceforge.OpenLieroX</id>
+  <metadata_license>CC0-1.0</metadata_license>
```
Comment (hfiguiere) [inline net.sourceforge.OpenLieroX.appdata.xml:4]: it's missing a project license

File: net.sourceforge.OpenLieroX.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    <screenshot>https://www.openlierox.net/official/screenshots/olx3.png</screenshot>
+    <screenshot>https://www.openlierox.net/official/screenshots/olx1.png</screenshot>
+  </screenshots>
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">intense</content_attribute>
+    <content_attribute id="violence-bloodshed">intense</content_attribute>
+    <content_attribute id="social-chat">intense</content_attribute>
+  </content_rating>
+  <releases>
+    <release version="0.58 rc5" date="2018-07-23"/>
```
Comment (hfiguiere) [inline net.sourceforge.OpenLieroX.appdata.xml:27]: I bet appstream validate will dislike the unordered releases.

File: net.sourceforge.OpenLieroX.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+app-id: net.sourceforge.OpenLieroX
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: openlierox
+rename-icon: OpenLieroX
+rename-desktop-file: openlierox.desktop
+finish-args:
+  - --socket=wayland
```
Comment (hfiguiere) [inline net.sourceforge.OpenLieroX.yaml:9]: are you sure it supports wayland since it uses SDL 1.2

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: net.sourceforge.OpenLieroX.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      - -DSYSTEM_DATA_DIR=/app/share
+      - -DHAWKNL_BUILTIN=ON
+      - -DLIBZIP_BUILTIN=ON
+    post-install:
+      - install -Dm755 bin/openlierox ${FLATPAK_DEST}/bin/openlierox
+      - install -d ${FLATPAK_DEST}/share/OpenLieroX/
+      - cp -r share/gamedir/* ${FLATPAK_DEST}/share/OpenLieroX/
+      - install -Dm644 share/OpenLieroX.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/OpenLieroX.svg
+      - install -Dm644 share/openlierox-openlierox.desktop ${FLATPAK_DEST}/share/applications/openlierox.desktop
+      - install -Dm644 net.sourceforge.OpenLieroX.appdata.xml ${FLATPAK_DEST}/share/appdata/openlierox.appdata.xml
```
Comment (hfiguiere) [inline net.sourceforge.OpenLieroX.yaml:53]: you install it with the wrong name. hence, it's not found.

**Comment by Mailaender** [comment]
bot, build net.sourceforge.OpenLieroX

**Comment by Mailaender** [comment]
bot, build net.sourceforge.OpenLieroX

**Comment by barthalion** [comment]
bot, build net.sourceforge.OpenLieroX

**Pull request closed without merging**